src/sparseml/pytorch/base.py

```diff
@@ -49,7 +49,7 @@
 
 
 _TORCH_MIN_VERSION = "1.0.0"
-_TORCH_MAX_VERSION = "1.13.100"  # set bug to 100 to support all future 1.9.X versions
+_TORCH_MAX_VERSION = "2.0.0"
```

Comment:
Actually building on Rahul's comment, we should match the patch version to .100 still to support future patches of 2.0.X

Reply:
I.e set `2.0.100` so that we can support all `2.0.x` versions in the future? @bfineran